pkg/standalone/standalone.go

```diff
@@ -151,6 +151,8 @@ func Init(runtimeVersion, dashboardVersion string, dockerNetwork string, slimMod
 		v1, v2 := parseVersionFile(fromDir)
 		if v1 != "" && v2 != "" {
 			runtimeVersion, dashboardVersion = v1, v2
+		} else {
+			return fmt.Errorf("runtime and dashboard versions cannot be parsed from version file in directory")
```

Comment:
If `parseVersionFile` returns error, it will be more helpful for the end user to fix the issue. Also, is it guaranteed to contain proper versioning? Or does that is something we should check as well.

Should we add a test to cover this?

Reply:
We need to add test for this whole feature. For the versions part ... it is part of the script for creating the installer-bundle. Please create a separate issue for adding e2e for this flag.